defaults/main.yml

```diff
@@ -78,17 +243,62 @@ pwm_activateuser_enable: false
 pwm_display_newuser_agreement: 'Terms and conditions of use'
 pwm_email_smtp_address: localhost
 pwm_email_newuser: 
-  - !unsafe '<value>{"to":"@User:Email@","from":"New User Registration &lt;@DefaultEmailFromAddress@&gt;","subject":"Welcome","bodyPlain":"Thank you for registering your account, @LDAP:givenName@.","bodyHtml":"&lt;b&gt;Thank you for registering your account, &lt;/b&gt;&lt;b&gt;@LDAP:givenName@.&lt;/b&gt;"}</value>'
-  - !unsafe '<value locale="es">{"to":"@User:Email@","from":"Registro  &lt;@DefaultEmailFromAddress@&gt;","subject":"Registro correcto","bodyPlain":"@LDAP:givenName@ gracias por su registro.","bodyHtml":"&lt;b&gt;@LDAP:givenName@ gracias por su registro.&lt;/b&gt;"}</value>'
+  - locale: en
+    to: "@User:Email@"
+    from: "New User Registration &lt;@DefaultEmailFromAddress@&gt;"
+    subject: "Welcome"
+    bodyPlain: !unsafe "Thank you for registering your account, @LDAP:givenName@."
+    bodyHtml: !unsafe "&lt;b&gt;Thank you for registering your account, &lt;/b&gt;&lt;b&gt;@LDAP:givenName@.&lt;/b&gt;"
```

Comment:
This was resolved as follows. Messages in HTML format (in their usual syntax) will be stored  as strings in variables to avoid generating multiple template files. These messages are translated into XML format using tasks. Also, variables with long texts could be defined in a separate site from the others variables (in the host host_vars). This tip should be in the role readme. I look forward to receiving the latest commits to approve the review.